src/lavinmq/client/client.cr

```diff
@@ -143,6 +145,11 @@ module LavinMQ
           if @running
             process_frame(frame)
           else
+            @closeok_timer += 1
+            if @closeok_timer > 500000
```

Comment:
we force close after 500 000 frames? i think that's a bit much, and what if they send frames very slowly? what about having a timer instead? say 5-10 seconds or so?